docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -0,0 +1,3023 @@
+
+If you are working through this tutorial at home, you can get the installers under the following links:
+
+- [OpenMS](https://www.openms.de/download/openms-binaries)
+- [KNIME](https://www.knime.org/downloads/overview)
+- OpenMS prerequisites (Windows-only): After installation, before your first use
+of the OpenMS plugin in KNIME you will be asked to download it automatically
+if certain requirements are not found in your Windows registry. Alternatively,
+you can get a bundled version here.
+
+Choose the installers for the platform you are working on.
+
+### Data conversion
+
+Each MS instrument vendor has one or more formats for storing the acquired data.
+Converting these data into an open format (preferably mzML) is the very first step
+when you want to work with open-source mass spectrometry software. A freely available conversion tool is MSConvert, which is part of a `ProteoWizard` installation. All files
+used in this tutorial have already been converted to mzML by us, so you do not need
+to perform the data conversion yourself. However, we provide a small raw file so you
+can try the important step of raw data conversion for yourself.
+
+```{note}
+The OpenMS installation package for Windows automatically installs
+ProteoWizard, so you do not need to download and install it separately. Due
+to restrictions from the instrument vendors, file format conversion for most
+formats is only possible on Windows systems. In practice, performing the
+conversion to mzML on the acquisition PC connected to the instrument is
+usually the most convenient option.
+```
+To convert raw data to mzML using `ProteoWizard` you can either use MSConvertGUI (a
+graphical user interface) or `msconvert` (a simple command line tool).
+
+(Figure_1)=
+|![msconvertgui](../images/openms-user-tutorial/introduction/proteowizard.png)|
+|:--:|
+|Figure 1: `MSConvertGUI` (part of `ProteoWizard`), allows converting raw files to mzML. Select the raw files you want to convert by clicking on the browse button and then on Add. Default parameters can usually be kept as-is. To reduce the initial data size, make sure that the `peakPicking` filter (converts profile data to centroided data (see Fig. 2)) is listed, enabled (true) and applied to all MS levels (parameter ”1-”). Start the conversion process by clicking on the Start button.|
+
+Both tools are
+available in: `C: / Program Files / OpenMS-2.7.0 / share / OpenMS / THIRDPARTY / pwiz-bin`.
+
+You can find a small RAW file on the USB stick `C: / Example_Data Introduction
+datasets/raw`.
+
+#### MSConvertGUI
+
+`MSConvertGUI` (see <a href="#figure-1">Fig. 1</a>) exposes the main parameters for data conversion in a convenient graphical user interface.
+
+#### msconvert
+
+The `msconvert` command line tool has no user interface but offers more options than the application `MSConvertGUI`. Additionally, since it can be used within a batch script, it allows converting large numbers of files and can be much more easily automatized.
```

Comment:
```suggestion
The `msconvert` command line tool has no graphical user interface but offers more options than the application `MSConvertGUI`. Additionally, since it can be used within a batch script, it allows converting large numbers of files and can be much more easily automatized.
```